### PR TITLE
Migrate to pandoc

### PR DESCRIPTION
**Pull request by weiznich**
I would prefer to keep the old layout, therefore I've used your branch to build a pandoc based version instead. A demo version should hopefully appear [here](https://weiznich.github.io/diesel.rs-website/index.html) soon.

Edit: One open point is to go through all the links and check them. Also I should have a look at how to keep the old page links in place. (They could simply forward to the new locations…)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Webpage
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y pandoc python3-pandocfilters make
+        pandoc --version
+        make page
+    - name: Deploy
+      if: github.repository == 'weiznich/diesel.rs-website' && github.event_name == 'push'
```
Comment (weiznich) [inline .github/workflows/ci.yml:21]:
```suggestion
      if: github.repository == 'sgrif/diesel.rs-website' && github.event_name == 'push'
```

Note to myself: Change this before merging the PR.

**Comment by JohnTitor** [comment]
Oh, looks better than mdBook to me! Seems your gh-page returns 404 except for the top page but otherwise, I'd follow this change 👍

**Comment by JohnTitor** [comment]
Hmm, now the main page's style is broken for me (https://weiznich.github.io/diesel.rs-website/index.html):

image

And the last guide description isn't displayed correctly (https://weiznich.github.io/diesel.rs-website/guides/index.html):

image

**Comment by JohnTitor** [comment]
Btw, we cannot mention an org team here, you could bring this to the (team's) discussion page if you want.

**Comment by weiznich** [comment]
@JohnTitor Those issues are fixed now.

cc @Ten0 @pksunkara

**Review by JohnTitor — APPROVED** [review]
Awesome, LGTM.

File: src/guides/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+<header class = "aside__header">A note on Rust versions:</header>
+
+::: aside__text
+Diesel requires Rust 1.31 or later. If you're following along with this guide,
+make sure you're using at least that version of Rust by running `rustup update stable`.
+
+:::
+</aside>
+
+## Install `diesel_cli`
```
Comment (Ten0) [inline src/guides/getting-started.md:30]: We seem to be doing more than just installing the Diesel CLI in this section

**Review by Ten0 — COMMENTED** [review]
No opposition towards the idea in general, however this seems to have a quite large impact on the visuals at this point, with at least the following:
1. Index page is less readable:
image
=>
image

2. Syntax highlighting is broken/damaged:
image
image
=>
image
image

3. The font is a bit wider, which I personally don't feel makes it more readable.

**Review by pksunkara — COMMENTED** [review]
Didn't find any glaring issue other than what's already commented.

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,2 @@
 .DS_Store
-/.bundle
-/.cache
-/.sass-cache
-/build
+/book
```
Comment (JohnTitor) [inline .gitignore:2]:
```suggestion
```

Well, we could remove this.

**Comment by weiznich** [comment]
I've pushed a new version that hopefully should fix the first two points mentioned by @Ten0. Please not that the syntax highlighting remains a bit different, as the highlighting itself is done differently. Hopefully the colors should be now more readable.
The third point does not happen for me, so it would be great to get some screenshot or better the corresponding font definitions from your web browser for the corresponding css block for this.

**Comment by weiznich** [comment]
If there are no other objectives I will merge this PR tomorrow.

**Review by Ten0 — APPROVED** [review]

File: src/guides/getting-started.md (hunk 2 above)
Comment (Ten0) [reply src/guides/getting-started.md:30]: @weiznich I'm not sure whether you've seen this comment

Comment (Ten0) [reply src/guides/getting-started.md:30]: This is content that was added as part of this PR, as there was no title in the guide before.

Comment (weiznich) [reply src/guides/getting-started.md]: Thanks for the comment. I've missed this change, as I took the md files from @JohnTitor branch 🙈 
I've introduced another headline there, to make it a bit more clear what happens where.

Comment (JohnTitor) [reply src/guides/getting-started.md]: Oh, seems I introduced it, thanks for catching up!

**Comment by weiznich** [comment]
I've deployed this version to the webpage, so it's live now 🎉

**Pull request merged**